server/backend/api/app/crud/crud_utils.py

```diff
@@ -87,12 +94,23 @@ def is_app_admin(
     if not application:
         error_msg = f"Application ID {application_id} not found"
         raise_http_exception(
-            error_msg=error_msg,
-            error_code=ERROR_CODE_INVALID_APPLICATION_ID
+            error_msg=error_msg, error_code=ERROR_CODE_INVALID_APPLICATION_ID
         )
 
     admin_role = f"{application.application_name.upper()}_ADMIN"
 
     if access_roles and admin_role in access_roles:
         return True
     return False
+
+
+def is_requester_external_delegated_admin(
+    db: Session,
+    requester: Requester,
+):
+    if (
+        requester.user_type_code == UserType.BCEID
+        and crud_access_control_privilege.is_delegated_admin(db, requester.user_id)
+    ):
+        return True
+    return False
```

Comment:
I was thinking this method could be in `crud_access_control_privilege` but it is fine to be here I think.